app/models/application_record.rb

```diff
@@ -11,4 +11,14 @@ def self.alias_error_messages(to_field, from_field)
   def error_message_aliases
     @@error_message_aliases
   end
+
+  def locking_enabled?
+    !!(super && @locking_enabled)
+  end
+
+  attr_writer :locking_enabled
+
+  def stale?
+    try(:lock_version_changed?) || false
```

Comment:
`|| false` to make sure this is a boolean?
If you always want a boolean, I'd add !!, with this, if try(:lock_version_changed?) is an integer > 0 or something, you aren't always getting back a boolean - I guess this prevents getting back a `nil`?

Reply:
Ah, good call.  I'm always inconsistent about that.